compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs

```diff
@@ -204,16 +206,15 @@ getSrcRoot fileOrDir = do
           pure $ toNormalizedFilePath root
 
 -- | Merge several packages into one.
-mergePkgs :: [WhnfPackage] -> LF.Package
-mergePkgs [] = error "No package build when building dar"
-mergePkgs (WhnfPackage pkg0:pkgs) =
-    foldl
-        (\pkg1 (WhnfPackage pkg2) ->
+mergePkgs :: LF.Version -> [WhnfPackage] -> LF.Package
+mergePkgs ver pkgs =
```

Comment:
I wonder if it was cleaner (possible?) to keep the empty match and simply return something "empty" and leave the rest.

Reply:
It definitely would be possible but I find it easier to understand and cleaner this way. It makes it clear that this is just a left-fold with the empty package being the identity.